src/control/lib/spdk/src/nvme_control.c

```diff
@@ -182,18 +180,17 @@ wipe_ctrlr(struct ctrlr_entry *centry, struct ns_entry *nentry)
 					    4096 / sector_size /** #LBAS */,
 					    write_complete, &data, 0);
 		if (rc != 0) {
-			snprintf(res->info, sizeof(res->info),
-				 "spdk_nvme_ns_cmd_write() (%d)\n", rc);
-			res->rc = -1;
+			snprintf(res->info, sizeof(res->info), "spdk_nvme_ns_cmd_write() (%d)\n",
+				 rc);
```

Comment:
If I'm understanding correctly how the errors stack up, we probably shouldn't include the rc in the message at this level, right?

Reply:
correct